salt/metalk8s/kubeadm/init/control-plane/files/manifest.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    scheduler.alpha.kubernetes.io/critical-pod: ""
+  creationTimestamp: null
+  name: {{ name }}
+  namespace: "kube-system"
+  labels:
```

Comment:
We'll want to add more labels later, see e.g. the `registry` and `salt-master` `Pod`s. Tracking ticket in order.

Reply:
Ok, I opened an issue: #668